labs-ios-starter.xcodeproj/project.pbxproj

```diff
@@ -994,7 +998,7 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = VXUQXR6S56;
+				DEVELOPMENT_TEAM = 34P4RF23ET;
```

Comment:
Is this intentional? I would double-check this in project to see which it should be

Reply:
I was just running it on my phone to see if I was running into a simulator bug. I must've forgotten to change it back